x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/index.test.tsx

```diff
@@ -328,23 +328,37 @@ describe('when on the hosts page', () => {
         expect(statusHealth).not.toBeNull();
         expect(message).not.toBeNull();
       });
-      it('should not show any numbered badges if all actions are succesful', () => {
-        return renderResult.findByTestId('hostDetailsPolicyResponseAttentionBadge').catch(e => {
+      it('should not show any numbered badges if all actions are successful', () => {
+        const policyResponse = docGenerator.generatePolicyResponse(
+          new Date().getTime(),
+          HostPolicyResponseActionStatus.success
+        );
+        store.dispatch({
```

Comment:
I wonder if you should wrap this in an `.act()` in order to ensure it is closer to react's internal rendering.

Reply:
can do